packages/core/graphql/query.graphql

```diff
@@ -0,0 +1,47 @@
+query statusByCid($dataCid: String!) {
+  requests: allStorageRequests(
+    condition: {dataCid: $dataCid}
+  ) {
+    nodes {
+      status
+      id
+      cid: dataCid
+      batch: batchByStoragePayloadId {
+        payload: storagePayloadByStoragePayloadId {
+          deals: dealsByStoragePayloadId {
+            nodes {
+              deal_id: dealId
+              miner: storageProviderId
+              deal_expiration: dealExpiration
+              status
```

Comment:
Just making sure this is on purpose... This `status` is our simple status value, not the filecoin market deal status.

Reply:
good catch, fixed